zbus_xmlgen/src/main.rs

```diff
@@ -25,6 +25,7 @@ fn usage() {
   zbus-xmlgen <interface.xml>
   zbus-xmlgen --system|--session <service> <object_path>
   zbus-xmlgen --address <address> <service> <object_path>
+  zbus-xmlgen <interface> --split-interfaces
```

Comment:
* This should be `<interface.xml>`, shouldn't it? 
* Does `--split-interfaces` not work with `--system|--session|--address` options? :thinking: This message gives the impression that it doesn't.

Reply:
The option works with all arguments, which is why I set it to `<interface>`. I didn't want to add it to the end of the other options, in fear someone would interpret it as required :)

Reply:
`<interface>` is opaque and doesn't tell the user what it means. My first guess as a user would be that you have to specify an interface name, which is wrong.